careers/jobs/senior-fullstack-node-engineer.md

```diff
@@ -0,0 +1,65 @@
+# Senior Fullstack Node Engineer
+
+## About DVELP
```

Comment:
I think we need to add in a bit more about the `DVELP Magic`. 

Below is the original paragraph from the very first job invite I created. Have a read and think if we should amend it to reflect where we are now and what you as a DVELPr believe in!

At Dvelp, we are about loving what we do. We love to learn, we love to experiment and we love to make the best products we can with the best tools we can find. We’re brave, we work hard and at our core we are about quality.

We are building an environment where we can work hard and sleep easy, where we can work remotely, but as part of a community, to deliver products globally.

We want to be challenged, be encouraged to ‘dare’, to think openly and not be restrained by corporate policy. We want to be better.

We are looking to work with an awesome front-end dev, who’s willing to get involved in the short-term, with the possibility of becoming a set piece of the team.

Reply:
I love that! I've inserted most of it into this section.